sdk/storage/src/datalake/path_client.cpp

```
@@ -47,6 +46,7 @@ namespace Azure { namespace Storage { namespace DataLake {
         {
           length = std::stoll(GetSubstringTillDelimiter('\n', subRangeString, cur)) - offset + 1;
         }
+        // else raise exception?
```

Yes, please throw an exception if parse failed

---

I'll throw an exception on this.